### PR TITLE
Update tfs-to-gccode.md

### PR DESCRIPTION
**Pull request by joel-ld-esdc**
Feel free to update or reformat my question @s-laugh

### CODE REVIEW

File: guides/tfs-to-gccode.md [hunk 1]
```diff
@@ -36,4 +36,7 @@ The platforms are diffrent engought that it could take more time to write the sc
 The other benifit of doing it manually is forcing your team to remove redundant and outdated information.
 (Don't move closed items.)
 
+### With the implementation of GitLab, we have to make our code open with the gouvernement. It also mean that we need to remove every bad habits like hardcoding secrets in code. We also need to remove every secrets from the web.config file. What are the recommended approachs for that ?
```
Comment (s-laugh) [inline guides/tfs-to-gccode.md]:
```suggestion
### What is the recommended approach for managing secrets in code? 
```

File: guides/tfs-to-gccode.md [hunk 2]
```diff
@@ -36,4 +36,7 @@ The platforms are diffrent engought that it could take more time to write the sc
 The other benifit of doing it manually is forcing your team to remove redundant and outdated information.
 (Don't move closed items.)
 
+### With the implementation of GitLab, we have to make our code open with the gouvernement. It also mean that we need to remove every bad habits like hardcoding secrets in code. We also need to remove every secrets from the web.config file. What are the recommended approachs for that ? 
+
+> When a new developper join an application, he will fork a solution from the repository. What would be the best way to insert all the necessary information in the web.config in order to make the application works. (i.e. : inserting secrets, server name, username for WSED, etc)
```
Comment (s-laugh) [inline guides/tfs-to-gccode.md]:
```suggestion
> When migrating to an open source platform (like GitHub or GCcode) we can't keep hardcoding secrets, and they need to be removed.
> How should a new developer get the necessary information to input into the web.config? (i.e. : inserting secrets, server name, username, etc)
```

**Comment by s-laugh** [comment]
Just a few suggestions to wording. Feel free to adjust further.

File: guides/tfs-to-gccode.md (hunk 1 above)
Comment (joel-ld-esdc) [reply guides/tfs-to-gccode.md]: It's great.

**Pull request merged**